templates/cluster-template-clusterclass.yaml

```diff
@@ -498,6 +503,13 @@ spec:
           port:
             type: integer
         type: object
+  - name: apiServerSigningCertExtraCertSANs
+    required: true
+    schema:
+      openAPIV3Schema:
+        description: Set extra Subject Alternative Names (SANs) for the API Server
+          signing certificate.
+        type: string
```

Comment:
this should be a list

Reply:
wanted to avoid any issue as mentioned in https://github.com/kubernetes-sigs/cluster-api/issues/6245